tests/utils/test_plotting.py

```diff
@@ -0,0 +1,118 @@
+from typing import List
+import rasa.utils.plotting
+import numpy as np
+import pytest
+
+
+@pytest.mark.parametrize(
+    "data, num_bins, expected_bins",
+    [
+        ([[1, 3, 8], [2, 3, 3]], 7, list(range(1, 9 + 1))),
+        ([[3, 8], [2, 3, 3]], 6, list(range(2, 9 + 1))),
+        ([[3, 7], [2, 3, 3]], 5, list(range(2, 8 + 1))),
```

Comment:
Is there a reason why the arguments inside `range` is specified as `9 + 1` and not `10`. If there is something subtle that you want to highlight with the offset `1`, then probably good to compliment it with a comment as well.